vernac/library.ml

```diff
@@ -265,7 +265,8 @@ module Intern = struct
     (** A pair of [kind, object], for example ["file",
         "/usr/local/foo.vo"], used for error messages. *)
   end
-  type t = DirPath.t -> library_t * Provenance.t
+
+  type t = DirPath.t -> (library_t, exn) Result.t * Provenance.t
```

Comment:
should it be exn or iexn?
I see we do Exninfo.info so it should probably be iexn so that we don't have to

Reply:
Thanks! Done.